specification/cognitiveservices/data-plane/CustomSearch/v1.0/examples/SuccessfulQueryRequest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "X-BingApis-SDK": "true",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "q": "cortana"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        
+      },
+      "body": {
+        "_type": "SearchResponse",
+        "webPages": {
```

Comment:
>webPages [](start = 9, length = 8)

Needs its discriminator

Reply:
What do you mean by discriminator? What do I need to add? This is the response to customsearch API copied over.

Reply:
https://github.com/Azure/oav/pull/171